test/services/ApprovalServiceSpec.scala

```diff
@@ -43,7 +43,7 @@ class ApprovalServiceSpec extends BaseSpec with MockFactory {
     val invalidProcess: JsObject = Json.obj("idx" -> invalidId)
```

Comment:
You could write a test with a simple process with the pass phrase flag and then test the process submitted to the repository save has the additional pass phrase stanzas.

Reply:
There is one in SecuredProcessBuilderSpec

Reply:
Yes, I see it